### PR TITLE
Explicit vae mnist

### PR DESCRIPTION
**Pull request by mcabbott**
This previously didn't have a manifest

### CODE REVIEW

File: vision/vae_mnist/Project.toml [hunk 1]
```diff
@@ -7,12 +7,12 @@ ImageIO = "82e4d734-157c-48bb-816b-45c225c6df19"
...
-Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
 TensorBoardLogger = "899adc3e-224a-11e9-021f-63837185c80f"
 
 [compat]
-CUDA = "2.4"
-Flux = "0.11.4, 0.12"
-julia = "1.5"
+CUDA = "2.4, 3.12"
```
Comment (ToucheSir) [inline vision/vae_mnist/Project.toml:15]:
```suggestion
CUDA = "3.12"
```
What is 3.12 needed for? Just the latest version?

**Review by ToucheSir — APPROVED** [review]
LGTM modulo conflicts. We'll have to clean up those `Encoder`/`Decoder` definitions at some point...

File: vision/vae_mnist/Project.toml (hunk 1 above)
Comment (mcabbott) [reply vision/vae_mnist/Project.toml:15]: Just the latest. Flux current requires 3.

**Comment by mcabbott** [comment]
I did check that adding types to `struct Encoder` has no effect on speed, BTW.

**Pull request merged**